### DISCUSSION TITLE
How to force open dialog in DiaisyUI v3.0.0?

### DISCUSSION THREAD

**Discussion by keanesnow98** [saadeghi/daisyui#1882] [Q&A]
I am really glad to hear about daisyUI version upgrade to 3!
So, I want to upgrade my project too.
![image](https://github.com/saadeghi/daisyui/assets/119104736/1aa56684-a3b8-42ac-9302-d69054ae6d72)
I copied this code except the first line.
Instead, I want to open dialog using javascript in somewhere else.
Is it possible to do so?

**Comment by BjarneBetjent** [comment]
Use a reference to the dialog to open it from wherever you want by calling `element.showModal()`

The docs got examples on how to do this in vanilla javascript
[showModal docs](https://developer.mozilla.org/en-US/docs/Web/API/HTMLDialogElement/showModal)
